http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -838,8 +828,10 @@ class DataprocInterpreter[F[_]: Parallel](
       isMember <- checkIsMember
       _ <- (isMember, addToGroup) match {
         case (false, true) =>
+          // Sometimes adding member to a group can take longer than when it gets to the point when we create dataproc cluster.
+          // Hence add polling here to make sure the 2 service accounts are added to the image user group properly before proceeding
           logger.info(ctx.loggingCtx)(s"Adding '$memberEmail' to group '$groupEmail'...") >>
-            addMemberToGroup
+            addMemberToGroup >> waitUntilMemberAdded(memberEmail)
```

Comment:
move wait here so that we're only waiting if the "adding member" action actually happened